CHANGELOG.adoc

```diff
@@ -10,6 +10,7 @@ Release with new features and bugfixes:
 * https://github.com/devonfw/IDEasy/issues/1243[#1243]: Fix display of ide status although errors might occur
 * https://github.com/devonfw/IDEasy/issues/1233[#1233]: Fix dropdown width in MSI installer for drive selection
 * https://github.com/devonfw/IDEasy/issues/1229[#1229]: Installation fails if version not available for current OS
+* https://github.com/devonfw/IDEasy/issues/1190[#1190]: Add pycharm support to IDEasy
```

Comment:
needs to moved up to the next release. Sorry.

Reply:
Adjusted.